bin/blendid.js

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env node
+const path = require('path');
+
 const additionalArgs = require('minimist')(process.argv.slice(2))._
+const blendidEntryFile = require.resolve('blendid');
+const gulpModulePath = path.dirname(require.resolve('gulp'));
+const gulpBinaryFile = path.join(gulpModulePath, '/bin/gulp');
 
-let args = ['--gulpfile', 'node_modules/blendid/gulpfile.js']
+let args = ['--gulpfile', blendidEntryFile]
```

Comment:
this seems to only be referencing `node_modules/blendid/`, a folder, not a file. Does gulp automatically search that folder for a gulpfile? Should this be renamed `blendedEntryPath`?

Reply:
Just checked the docs - looks like this is correct.